src/middle-earth/comparisons.py

```diff
@@ -0,0 +1,103 @@
+from math import sqrt
```

Comment:
Related to the other suggestion.
```suggestion
from math import hypot, sqrt
```